storage/remote/otlptranslator/prometheusremotewrite/helper_test.go

```diff
@@ -506,7 +506,7 @@ func TestCreateAttributes(t *testing.T) {
 					PromoteResourceAttributes:    tc.promoteResourceAttributes,
 					IgnoreResourceAttributes:     tc.ignoreResourceAttributes,
 				}),
-				ConvertScopeMetadata: tc.convertScope,
+				PromoteScopeMetadata: tc.convertScope,
```

Comment:
nit: consider adjusting this test as well

```suggestion
				PromoteScopeMetadata: tc.promoteScope,
```